nasa/harmony
# Harmony 1537 env vars

## Jira Issue ID
HARMONY-1537

## Description
Refactor env vars and add validation on startup to make sure the various environments are set up correctly / have all the env vars they need.

## Local Test Steps
1. check out this branch
2. Run `npm install`
3. Run `npm test`
4. Test Harmony in a Box by running `bin/bootstrap-harmony` and trying a query
5. Optionally do a sandbox deployment to and make sure things work

## PR Acceptance Checklist
